src/python_inspector/resolve_cli.py

```diff
@@ -172,6 +172,7 @@ def print_version(ctx, param, value):
     callback=print_version,
     help="Show the version and exit.",
 )
+@click.option('--permissive', is_flag=True, default=False, help='Enable permissive mode.')
```

Comment:
May be we can find a better name for this? "permissive" seems to be related to permissions... but the concept here is more: keep trucking on error, do not fail on error

Reply:
Renamed to --ignore-errors :+1: